content/support/changelog/ap/index.md

```diff
@@ -14,9 +14,62 @@ title: Accounts Payable Changelog
 
 Monitor this page to keep up with the [Accounts Payable API]({{ stache.config.portal_endpoints_AP }}) latest changes and {{ stache.config.api_type_name }} service releases.
 
-## 2018-04-20
+## 2018-06-01
 
-### Changed
+### New
+
+Added the following endpoint:
+
+<div class="table-responsive">
+	<table class="table table-striped table-hover">
+		<thead>
+			<tr>
+				<th>Operation</th>
+				<th>Method</th>
+				<th>Route</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr class="clickable-row" data-url="{{ stache.config.portal_endpoints_invoice_edit_patch }}">
+				<td>Invoice edit</td>
+				<td>PATCH</td>
+				<td>/invoices/{invoice_id}</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+With the new endpoint, note the following:
+
+- You can edit the following values of an existing invoice:
+            - `vendor_id`
+            - `vendor_name`
+            - `invoice_number`
+            - `invoice_date`
+            - `due_date`
+            - `balance`
+            - `description`
+            - `status`
+            - `invoice_payment_details`
+            - `post_status`
+            - `post_date`
+            - `distribution_discounts`
+            - `distributions`
+            - `custom_fields`
+- Posted and paid invoices cannot be edited.
```

Comment:
I'd change this to use active voice: "You cannot edit posted and paid invoices." ... Also, in the next bullet, "Posted" and "Paid" are capitalized code snippets, but here they are just normal adjectives. I think that in both cases we're describing the same property, so it seems like we should consistently either format or not format. I'm not sure which though. @

Reply:
We try to avoid "You can't" and "Don't..." in our SKY voice/style, but that's more for error messages I think. For this, I'll change to active voice. I'll update the format too.